### PR TITLE
Your outfit.jsx with localStorage

### PR DESCRIPTION
**Pull request by cwg83**

### CODE REVIEW

File: client/src/components/App.jsx [hunk 1]
```diff
@@ -9,7 +9,7 @@ import '../style.css';
 const App = () => {
   const params = useParams();
   const id = params.id;
-  const [yourOutfit, setYourOutfit] = useState([]);
+  const [yourOutfit, setYourOutfit] = useState(JSON.parse(localStorage.getItem('yourOutfit')));
```
Comment (cwg83) [inline client/src/components/App.jsx:12]: Now the yourOutfit state will will be automatically set from localStorage (data saved in the user's browser) and adding / removing from yourOutfit will also update localStorage.

Comment (cwg83) [reply client/src/components/App.jsx:12]: This allows the user's outfit to be saved after existing and re-opening the website. You can look at localStorage by going to dev tools -> Application -> Local Storage as seen here:
[https://i.imgur.com/wDXr30z.png](https://i.imgur.com/wDXr30z.png)

Comment (sm-fm) [reply client/src/components/App.jsx:12]: This looks great! I'm assuming I should change mine to also just put the product in localStorage as well?

**Review by sm-fm — COMMENTED** [review]
Looks solid!

File: client/src/components/App.jsx (hunk 1 above)
Comment (cwg83) [reply client/src/components/App.jsx:12]: You won't need to, as the addToOutfit function in App.jsx will handle that!

**Pull request merged**